storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -7375,6 +7589,10 @@ int main(int argc, char **argv)
 	}
 #endif
 
+	if (opt_check_privileges) {
```

Comment:
IMO, `xtrabackup_backup_func` is big enough, and I would prefer not to extend it any further.